lib/ui/playground/playground_take_image_tester.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:io';
+
+import 'package:farmsmart_flutter/ui/common/take_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:image_picker/image_picker.dart';
+
+class _Strings{
+  static final noImage = 'No image';
+}
+
+class PlaygroundTakeImageTester extends StatefulWidget {
+  final ImageSource imageSource;
+
+  PlaygroundTakeImageTester({Key key, this.imageSource});
+
+  @override
+  _PlaygroundTakeImageTesterState createState() =>
+      _PlaygroundTakeImageTesterState();
+}
+
+class _PlaygroundTakeImageTesterState extends State<PlaygroundTakeImageTester> {
+  File _file;
+
+  @override
+  void initState() {
+    super.initState();
+
+    TakeImage(
```

Comment:
what about a static helper that turn the use into a one liner
ImagePicker.pickImage( success { }, onError { });

Reply:
Yep much better

Reply:
Done